pkg/doc/did/doc.go

```diff
@@ -423,6 +599,11 @@ func decodePK(rawPK map[string]interface{}) ([]byte, error) {
 func validate(data []byte) error {
 	// Validate that the DID Document conforms to the serialization of the DID Document data model.
 	// Reference: https://w3c-ccg.github.io/did-spec/#did-documents)
+	schemaLoader := schemaLoaderV1
+	if strings.Contains(string(data), contextV011) {
```

Comment:
May be better to check the context value `contextV011` against actual `context` key inside data rather than whole document/json ? we can avoid cases where the context is not `contextV011` but some other attribute in the json contains the text `contextV011`.

Reply:
Agree.

Reply:
done